content/material_and_questions.md

```diff
@@ -44,4 +53,153 @@ In the below table, any minimum or maximum variation between material years are
+Must have more than one answer.
+- The clarification of a single answer does not meet the above requirement.
+- The identification of a singular pronoun does not meet the above requirement.
+- The identification of a plural pronoun does meet the above requirement.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+If the multiple answers contain at least one "positive" answer AND at least one "negative" answer, this is deemed invalid as a Multiple Answer question.
+The Question Type abbreviation is "MA"
+
+#### Reference
+Used to distinguish exact same words or phrases from the material.
+The entire question is part of the required question + answer.
+If a quizzer needs to provide the reference question, the provided reference question must: 
+- Be of the exact same type as the one on the card (CR, CVR, CRMA, CVRMA).  The quizmaster and/or answer judge must check to confirm this.
+- Have content AND wording is similar to the content and wording of the written question.
+    - For greater clarity, we have provided the following example: CVR According to Luke chapter 23: 37 King of what?  A. The Jews The quizzer can give the question “King of what?” or “King of whom?”, as Jews could be referring to either nation (what) or people (whom).  In this case the quizzer could also give “King of the what?”
+- Have the determining word or phrase from the quizmaster’s card. 
+    - In the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar.
+The determination of a valid Reference question, and the valid sub-type only considers the content in the question. 
+
+##### Chapter-Verse
+Must have a single answer.
+The question must contain a word or phrase that exists more than once in the same chapter.
+Context is the single verse.
+The Question Type abbreviation is "CVR"
+
+##### Multiple-Answer, Chapter-Verse
+Must have more than one answer.
+The Question Type abbreviation is "MACVR"
+
+##### Chapter (only)
+The question must contain a word or phrase that exists more than once in the material.
+The Question Type abbreviation is "CR"
+
+##### Multiple-Answer, Chapter (only)
+Must have more than one answer.
+The Question Type abbreviation is "MACR"
+
+#### Quote
+Will be direct quotes from Scripture
+Must be quoted word-perfect.
+Must be strong enough to stand on its own without requiring additional verses to explain it.
+Must be spiritually significant.
+The entire reference is part of the required question + answer.
+Quizzers may go back and correct their answers if done in the 30-second time limit.  The quizzer will be required to make one full correct rotation. However, the quizzer can be called correct if they have made a mistake only in one area of the verse AND correct the mistake making it abundantly clear to the quiz master and answer judge that they know where their mistake was, and what he corrected words are.
+Context is defined as the verses or verses that are required.
```

Comment:
the *verse or verses that are required

Introducing the concept of "context" into Finish/Quote questions changes the categories of the original rulebook somewhat. If I'm reading it correctly, "context" isn't part of these questions; you merely have the rule that says "quizzer may not quote from any but the required verse or verses." I don't think there's a substantive change here, but we might want to avoid rephrasing where it's not helpful or necessary. When you think about it, "context" isn't really the right word for what this rule is about. (Of course, you have the problem with CVRs having "one verse" of context as well, but it makes sense to define those that way because we need to distinguish it from the 5-verse context of CRs.)

Reply:
I totally see what you're saying, but why on earth would we have the notion of "context" AND the notion of "must stay in the verse" ??

and if read literally, I can absolutely say a a single word took a quizzer into a different verse now (The Lana Internationals scenario). The language about one word not taking you out of context doesn't apply. Right?

Reply:
Logically, I can see it from both sides. But I'm thinking it'll be less confusing to define the *concept* of context, then change the *size* of context based on question type.

Reply:
The "one word can't take you out of context" was applied by an Internationals QM to an F. So it's pretty clear people think of "must stay in the verse" as synonymous with the notion of "context"

Reply:
@gryphonshafer agrees with me here, so technically I can resolve this comment thread. But I want to hear from @jswingle again, before we "convert" "stay in the same verse" to the concept of "context."

Reply:
I like the idea of changing the size of context relative to the question type. I think that is much more clear and allows for there to be "Context Rules" and not just a "context rule." That would allow them to be easily gathered into a single place if desired.